config/environments/production.rb

```diff
@@ -64,6 +64,20 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :smtp
+
+  config.action_mailer.perform_deliveries = true
+
+  config.action_mailer.smtp_settings = {
+    user_name: ENV["SENDGRID_USERNAME"],
+    password: ENV["SENDGRID_PASSWORD"],
+    domain: "PROJECT_DOMAIN",
```

Comment:
As I recall, domain is an optional setting.
If set, it should be set with a `ENV` since it will be different in the various environments.